src/containers/SearchPage/components/results/SearchConcept/ConceptForm.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useEffect, useContext } from 'react';
+import isEqual from 'lodash/fp/isEqual';
+import { injectT, tType } from '@ndla/i18n';
+import { useFormik } from 'formik';
+import { Select } from '@ndla/forms';
+import Button, { MultiButton } from '@ndla/button';
+import { DRAFT_PUBLISH_SCOPE } from '../../../../../constants';
+import { UserAccessContext } from '../../../../App/App';
+import { fetchSearchTags } from '../../../../../modules/draft/draftApi';
+import AsyncSearchTags from '../../../../../components/Dropdown/asyncDropdown/AsyncSearchTags';
+import { MultiSelectDropdown } from '../../../../../components/Dropdown/MultiSelectDropdown';
+import { SubjectType } from '../../../../../interfaces';
+import { InputField, InputPair } from './SearchStyles';
+
+export interface License {
+  description: string;
+  license: string;
+  url?: string;
+}
+
+export interface ConceptFormType {
+  title: string;
+  author: string;
+  license?: string;
+  subjects: SubjectType[];
+  tags: string[];
+  newStatus?: string;
+}
+
+interface Props {
+  initialValues: ConceptFormType;
+  status: string;
+  language: string;
+  onSubmit: (c: ConceptFormType) => void;
+  licenses: License[];
+  allSubjects: SubjectType[];
+  cancel: () => void;
+}
+
+const ConceptForm = ({
+  initialValues,
+  status,
+  language,
+  onSubmit,
+  licenses,
+  allSubjects,
+  cancel,
+  t,
+}: Props & tType) => {
+  const formik = useFormik<ConceptFormType>({
+    initialValues,
+    onSubmit,
+  });
+  const { values, handleChange, setValues } = formik;
+  useEffect(() => {
+    setValues({ ...values, title: initialValues.title });
+  }, [initialValues.title]);
+  const userAccess = useContext<string>(UserAccessContext);
+  const canPublish = userAccess.includes(DRAFT_PUBLISH_SCOPE);
+  const hidePublishButton =
```

Comment:
Eg ville tru at du ikkje trenger sjekk på status her for om den som endrer forklaringa har rettigheter til det burde han få lov å lagre og publisere i en operasjon. Ellers må han først lagre og så publisere.

Reply:
Det var ikke slik jeg tolket samtalen med kunden, jeg må vel uansett gjøre en sjekk på status, da ikke alle statuser kan føre til "publisert"

Reply:
Sant. Dessuten så så det ut til å fungere slik eg ønska. At om eg gjorde endringer så kunne eg lagre og publisere.